services/apiService.ts

```diff
@@ -112,3 +112,12 @@ export const getPendingBoostClaims = async (addr: string) => {
     console.log(err);
   }
 };
+
+export const getBoostedQuests = async () => {
+  try {
+    const response = await fetch(`${baseurl}/get_boosted_quests`);
+    return await response.json();
+  } catch (err) {
+    console.log(err);
```

Comment:
Same here. 

IF possible change it everywhere

Reply:
done

Reply:
Then can you delete the `console.log(err)` here and everywhere you find it? 

Cause you print two times now lol.

Reply:
fixed